### DISCUSSION TITLE
Audio Issue

### DISCUSSION THREAD

**Discussion by hilfam** [86Box/86Box#4136] [General]
When 86Box plays audio, if there is any hard disk activity of any kind, the audio "ticks" incessantly. This has been a problem since I started using 86Box, and it happens on multiple different platforms. I love 86Box, but the audio feature is very glitchy.

**Comment by hilfam** [comment]
This problem persists when using PCem also.

**Comment by TechGuyC** [comment]
> When 86Box plays audio, if there is any hard disk activity of any kind, the audio "ticks" incessantly. This has been a problem since I started using 86Box, and it happens on multiple different platforms. I love 86Box, but the audio feature is very glitchy.

Maybe use a lower speed processor? in my pc,emulation of Pentium 2 450mhz is Laggy

**Reply by hilfam** [reply]
I have tried multiple different processors, and if I remember correctly, the issue persists as far back as a 386 processor.

**Comment by eddmanx** [comment]
What's your computer's CPU and OS?
Which exact version of 86box are you using?
Also post the 86box machine config that has the audio issue.

**Reply by hilfam** [reply]
I'm using a Dell laptop with an Intel Core i5-1035G1. It is running Windows 11 23H2. The 86Box version is 4.0.1 build 5111 for Windows 64 bit. I've had this issue on multiple 86Box machines, I'd say maybe 10 different machines running different operating systems.

**Reply by eddmanx** [reply]
Is the emulation speed 100%?

Have you checked the DPC latency? You can check it with Latencymon.

Maybe try a newer 86box build.

**Reply by hilfam** [reply]
The emulation speed is 100%, and the build I'm using is the latest stable release. I'll try using the newest nightly build.

**Reply by hilfam** [reply]
I checked the newest nightly build (build 5484), and the problem still persists.
